test/functional/apps/discover/_discover_accessibility.ts

```diff
@@ -86,6 +86,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await focusAndPressButton('discoverAlertsButton');
         expect(await hasFocus('discoverAlertsButton')).to.be(false);
         await focusAndPressButton('discoverCreateAlertButton');
+        await retry.waitFor(
+          'Create Rule flyout is visible',
+          async () => await testSubjects.exists('addRuleFlyoutTitle')
+        );
```

Comment:
It fixes https://github.com/elastic/kibana/issues/135305 , right?

Reply:
Yes I think it will. I didn't start getting failures locally until after these changes, but it's the same error I was getting.